orderly_web/config.py

```diff
@@ -95,6 +184,20 @@ def __str__(self):
         return "{}/{}:{}".format(self.repo, self.name, self.tag)
 
 
+def config_data_update(path, data, extra=None, options=None):
+    data = copy.deepcopy(data)
+    if extra:
+        data_extra = read_yaml("{}/{}.yml".format(path, extra))
+        if "container_prefix" in data_extra:
+            raise Exception("'container_prefix' may not be modified")
+        data = combine(data, data_extra)
+    if options:
+        data = combine(data, options)
```

Comment:
This does seem to be passing in the original list of dictionaries to the combine method rather than a single dict. 
I ran: orderly-web start config/basic --option=proxy.enabled=false
and got exception:
 _File "/home/emma/.local/lib/python3.6/site-packages/orderly_web-0.0.1-py3.6.egg/orderly_web/config.py", line 264, in combine
    for k, v in b.items():
AttributeError: 'list' object has no attribute 'items'_

Reply:
Thanks, fixed in 5611c1e